.github/renovate.json5

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:best-practices",
```

Comment:
Do we want to use [`best-practices`](https://github.com/renovatebot/renovate/pull/23773/files) or `recommended`?

The current file is different than what we have in https://github.com/open-telemetry/sig-security/blob/main/.github/renovate.json5.

Reply:
+1 on best-practices. It has more explicit and exact versioning for GitHub actions and more in-depth dependency versioning.